operator/standalone/create.go

```diff
@@ -213,6 +213,10 @@ func (p *CreateStandalonePipeline) applyValuesFromInstance(_ context.Context) er
 			"persistence": helmvalues.V{
 				"size": p.instance.Spec.Parameters.Resources.StorageCapacity.String(),
 			},
+			"podAnnotations": helmvalues.V{ // these annotations can stay, even if backups are disabled.
+				"k8up.io/backupcommand":  `sh -c 'PGUSER="postgres" PGPASSWORD="$POSTGRES_POSTGRES_PASSWORD" pg_dumpall --clean'`,
```

Comment:
What happens if I disable root user postgres from postgresstandalone CRD?

Reply:
Then backups don't work :see_no_evil: 
You're right though, that feature is now conflicting...

I think we'll have to always enable postgres superuser, but maybe we just don't expose it in the connection secret in case the user disabled it?
When thinking ahead to the point where we can support multiple users and databases, it'll become imperative that we backup all databases. Then we'd definitely need a user that can access all databases.

So I think for simplicity reasons, we'll always enable the superuser regardless what the user specified, but we only expose the superuser credentials when the user enabled it. WDYT?

Reply:
THis proposal is almost good but there's a problem with it. If I created a postgresstandalone CRD with super user enabled, then I disable it via an update then I have access to the user anyway if I saved the password! What I wuld propose is to create a backup user for dumps.... it's the most sensible way to move forward instead of playing with the default admin account!

Reply:
True, I haven't thought of that...
However, creating a backup user and ensuring this user has access to all databases is going to be hard to implement with considerable effort. (see the notes in https://www.postgresql.org/docs/current/app-pg-dumpall.html, option `--role=rolename`).

Can we maybe still go with my proposal to use postgres as the backup user, and also create a new issue that tracks this technical debt? Otherwise this requires extra code to create and maintain the backup user, which I don't really want to do now. I mean, If a legitimate user wants to access the instance with superuser, they can do just that be re-enabling the super user again, not need to save the password anyway :shrug: So I don't think this is a big issue.

Reply:
As agreed internally, we'll not engineer a backup user at this moment for the specific edge case and go with the superuser account. We'll track this case as a technical debt in another issue, and it should be looked at at latest when engineering multi-user and -db support for instances.